pulpcore/pulpcore/app/serializers/base.py

```diff
@@ -259,7 +259,7 @@ class Meta:
     as-needed.
 
     """
-    type = serializers.CharField(read_only=True)
+    type = serializers.CharField(read_only=True, allow_blank=True)
```

Comment:
allow_blank isn't needed for the read_only fields, since we would never write to them